shell.go

```diff
@@ -6,8 +6,8 @@ import (
 	"fmt"
 	"io"
 	"strings"
-	"sync"
 
+	"github.com/google/martian"
```

Comment:
Is this dependency necessary? Upstream may want to avoid adding new dependencies.